### PR TITLE
[KK-0023]: 💄 feat: Changed the look of Navbar

### PR DESCRIPTION
**Pull request by kinga-kus**
# Description
In this PR I have changed the look of Navbar component and made it more responsive.

### Description and screenshots

I have also changed the look of the project in Figma and now this is how it looks Navbar rn:
https://www.figma.com/file/duToCPC6a9qlzfpcS7dGBi/Blood-Donation?node-id=271%3A11297&t=S5tTWYzQiUoC4m4R-4

This is how it look on a full screen - large screen:
Zrzut ekranu 2023-01-16 o 20 04 46

This is how it look on a medium and small screen:
Zrzut ekranu 2023-01-16 o 20 04 55

After clicking on menu icon:
Zrzut ekranu 2023-01-16 o 20 05 00

Hover on the "Become a donor" button:
Zrzut ekranu 2023-01-16 o 20 04 50

### CODE REVIEW

File: src/components/OnePage/Navbar.tsx [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
-}));
-
-const NavbarLinks = [
+const pages = [
   { label: 'Home', path: '/Blood-Donation' },
-  { label: 'O Nas', path: '/Blood-Donation/#mission' },
-  { label: 'Informacje', path: '/informacje' },
-  { label: 'Kontakt', path: '/Blood-Donation/#contact' }
+  { label: 'About Us', path: '/Blood-Donation/#mission' },
+  { label: 'Information', path: '/informacje' },
```
Comment (adam-kostuch) [inline src/components/OnePage/Navbar.tsx:26]: Why does all the paths lead to `/Blood-Donation/#...`, but the `Informacje` lead to simple `/informacje` and why is it written in Polish not English? 😃

File: src/components/OnePage/Navbar.tsx [hunk 2]
```diff
@@ -57,56 +69,172 @@ const NavbarLogo = styled('img')(({ theme }) => ({
...
+    setAnchorElUser(event.currentTarget);
+  };
+
+  const handleCloseNavMenu = () => {
+    setAnchorElNav(null);
+  };
+
+  const handleCloseUserMenu = () => {
+    setAnchorElUser(null);
+  };
```
Comment (adam-kostuch) [inline src/components/OnePage/Navbar.tsx:98]: Wouldn't it be possible to reduce this number of variables?

File: src/components/OnePage/Navbar.tsx (hunk 1 above)
Comment (kinga-kus) [reply src/components/OnePage/Navbar.tsx:26]: ohhhh i have forgotten about changing it, merci beaucoup! 🙏

File: src/components/OnePage/Navbar.tsx (hunk 2 above)
Comment (kinga-kus) [reply src/components/OnePage/Navbar.tsx]: yup

**Review by adam-kostuch — APPROVED** [review]

**Review by adam-kostuch — APPROVED** [review]

**Pull request merged**